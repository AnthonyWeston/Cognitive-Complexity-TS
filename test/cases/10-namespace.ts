namespace N {

}

namespace M {
    // nesting 0
    while (true); // +1
}
