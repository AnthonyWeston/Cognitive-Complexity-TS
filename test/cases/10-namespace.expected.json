{
    "10-namespace.ts": {
        "score": 1,
        "inner": [
            {
                "name": "N",
                "line": 1,
                "column": 1,
                "score": 0
            },
            {
                "name": "M",
                "line": 5,
                "column": 1,
                "score": 1
            }
        ]
    }
}